hub/policy/src/main/java/uk/gov/ida/hub/policy/domain/controller/AbstractMatchRequestSentStateController.java

```diff
@@ -80,55 +79,83 @@ public void handleRequestFailure() {
         stateTransitionAction.transitionTo(matchingServiceRequestErrorState);
     }
 
-    protected U getSuccessfulMatchState(MatchFromMatchingService responseFromMatchingService) {
-        String matchingServiceAssertion = responseFromMatchingService.getMatchingServiceAssertion();
-        validator.validate(responseFromMatchingService.getLevelOfAssurance(), state.getIdpLevelOfAssurance());
-        String requestIssuerId = state.getRequestIssuerEntityId();
-        return createSuccessfulMatchState(matchingServiceAssertion, requestIssuerId);
+    @Override
```

Comment:
Reordering of methods in a class is making it very hard to review. It's making the diff unnecessarily large...

Reply:
I sincerely repent